sensu.gemspec

```diff
@@ -34,6 +34,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency "em-http-request", "~> 1.1"
 
   s.files         = Dir.glob("{bin,lib}/**/*") + %w[sensu.gemspec README.md CHANGELOG.md MIT-LICENSE.txt]
-  s.executables   = Dir.glob("bin/**/*").map { |file| File.basename(file) }
+  s.executables   = Dir.glob("exe/**/*").map { |file| File.basename(file) }
```

Comment:
@bf4 cool, thank you for the clarification :+1: I have kicked the failed build, will merge this PR when it goes green! Thank you.